### ISSUE TITLE
"Where Ideas Work" banner

### ISSUE THREAD

**Issue by Travis-A-Clark** [bcgov/performance#607]
**Files**
![WIW_Logo.jpg](https://images.zenhubusercontent.com/61e98ff375d0505e601dd00e/63963370-e25a-4970-867d-a2014760f08b)

**Mockup**
![WIW Logo.jpg](https://images.zenhubusercontent.com/61e98ff375d0505e601dd00e/86a7b3ac-d461-4767-a181-df8b9679d5b3)

**Comment by jchow11** [comment]
Mockup attached! Files attached!

**Comment by Travis-A-Clark** [comment]
@jp-Telus Hoping you might have time next week to take care of some of these smaller fixes on the design and accessibility side.

**Comment by Travis-A-Clark** [comment]
@jp-Telus I was chatting with @telusdcinco about this; I think for now we should put the banner as a much smaller element at the bottom of the left hand navigation bar. Similar to the way we have the Performance Development Platform graphic at the top.

**Comment by jchow11** [comment]
@Travis-A-Clark I have created 2 options for the WIW Logo– my suggestion is to use the first option. 

**OPTION 1**
I separated "Where ideas work" and the BC Public Service logo in order to have the logo meet size requirements for web (70px height minimum). According to documentation, this seems to be allowed. 

![WIW Logo.jpg](https://images.zenhubusercontent.com/61e98ff375d0505e601dd00e/86a7b3ac-d461-4767-a181-df8b9679d5b3)
![Screen Shot 2022-07-13 at 10.07.30 AM.png](https://images.zenhubusercontent.com/61e98ff375d0505e601dd00e/7b1914d7-8d41-4acf-96eb-ce76db24fdf4)

**OPTION 2**
The width of the nav bar is not large enough to allow for the logo to be at least 70px in height. This option breaks this convention.
![WIW Logo 2.jpg](https://images.zenhubusercontent.com/61e98ff375d0505e601dd00e/c4f15854-4587-446f-90d7-f081ed6c53a5)
